### ISSUE TITLE
DOM error crashes browser tab

### ISSUE THREAD

**Issue by tre-dev** [marsprince/slate-vue#63]
## Replica

- Go to https://marsprince.github.io/slate-vue/#/markdown-shortcuts
- Select all text (cmd + a & cmd + c)
- Paste it (cmd + v)
- Paste it again => throws error in console and browser tab stops working
- Doing the same thing on react slatejs works: https://www.slatejs.org/examples/markdown-preview

## Error: 

```
Error: Cannot resolve a DOM node from Slate node: {"text":"Try it out for yourself! Try starting a new line with \">\", \"-\", or \"#\"s."}
    at Object.toDOMNode (main.js:2)
    at Object.toDOMPoint (main.js:2)
    at toDOMRange (main.js:2)
    at Object.ae.toDOMRange (main.js:2)
    at a.<anonymous> (main.js:2)
    at a.e (main.js:2)
    at qe (main.js:2)
    at a.e.$emit (main.js:2)
    at nn (main.js:2)
    at main.js:2
```

---

Not sure what triggered the error, but in any case, it would be great if errors could be handled so it doesn't result in the hangup of the whole site.

**Comment by marsprince** [comment]
Thank you for your feedback. Already under fixing in https://github.com/marsprince/slate-vue/issues/62 about paste problem.

**Comment by marsprince** [comment]
Fixed in v0.2.0

**Comment by tre-dev** [comment]
Thanks!
